### PR TITLE
More Admin Tools

### PR DESCRIPTION
**Pull request by ThatDan123**
-Start round, End round.

-Call shuttle, Recall Shuttle.

-Change Alert Level.

-Moved most admin commands to new manager, from player network actions.

-Commands should work in lobby, however the play global sound wont as the action finds playerscripts to play sounds on which dont exist in lobby, different system will be needed to play sounds for lobby users.

-Fixes Disposals NRE

### CODE REVIEW

**Comment by ThatDan123** [comment]
Gonna take some more time to do this.... damn authority.

File: UnityProject/Assets/Scripts/UI/PreRound/GUI_PreRoundWindow.cs [hunk 1]
```diff
@@ -77,9 +78,9 @@ private void OnDisable()
 
 	private void Update()
 	{
-		// TODO: remove once admin system is in
-		if (Input.GetKeyDown(KeyCode.F7) && !BuildPreferences.isForRelease)
+		if (Input.GetKeyDown(KeyCode.F7))
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/UI/PreRound/GUI_PreRoundWindow.cs:81]: does this still work if you're in the editor

Comment (ThatDan123) [reply UnityProject/Assets/Scripts/UI/PreRound/GUI_PreRoundWindow.cs:81]: Yup

Comment (ThatDan123) [reply UnityProject/Assets/Scripts/UI/PreRound/GUI_PreRoundWindow.cs:81]: It worked in builds too, so the build preferences.isForRelease didnt do anything anyway? So i removed it.

**Comment by ThatDan123** [comment]
DNM, theres one thing ive still gotta fix.

**Comment by ThatDan123** [comment]
Good to merge.

**Pull request merged**